2200021/14654.py

```diff
@@ -0,0 +1,17 @@
+# 신용카드 만들기 2
+
+import sys
+sys.stdin = open('input.txt','r')
+input = sys.stdin.readline
+
+for t in range(1,int(input())+1):
+    s = input()
+    li = ['3','4','5','6','9']
+    for i in s:
+        s = s.replace('-','')
```

Comment:
`replace` 함수 활용 잘해주셨어요! 👍 
```suggestion
    s = s.replace('-','')
```
`replace` 함수는 한 번에 모든 대상 문자열을 치환하기 때문에, 한 번만 써주셔도 됩니다!